resolver/resolver.go

```diff
@@ -72,7 +72,8 @@ func SetDefaultScheme(scheme string) {
 	defaultScheme = scheme
 }
 
-// GetDefaultScheme gets the default scheme that will be used.
+// GetDefaultScheme gets the default scheme that will be used, if no scheme was
+// overridden, defaults to the "dns" scheme.
```

Comment:
Maybe this can say `...the default scheme that will be used by grpc.Dial.  If SetDefaultScheme is never called, the default scheme used by grpc.NewClient is "dns" instead.`